CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1 @@
+This repository is governed by Mozilla's code of conduct and etiquette guidelines. For more details please see the [Mozilla Community Participation Guidelines] (https://www.mozilla.org/en-US/about/governance/policies/participation/) and [Developer Etiquette Guidelines] (https://bugzilla.mozilla.org/page.cgi?id=etiquette.html).
```

Comment:
Might be nice to add in a heading for consistency with other similar docs - e.g: https://github.com/mozilla/addons-server/blob/master/CONTRIBUTING.rst